docs/datavalidator/check_json.py

```diff
@@ -125,9 +220,25 @@ def check_collections(self) -> None:
             raise CheckException(err)
 
     def check_model(self, collection: str, model: Dict[str, Any]) -> None:
-        model_fields = set(x for x in model.keys() if "$" not in x)
+        errors = self.check_normal_fields(model, collection)
+
+        if not errors:
```

Comment:
Isn't it better to check everything, regardless if there were errors?

Reply:
The later checks rely on these checks to succeed. E.g. when checking relations, I can be sure, that all fields are there, I just have to check values.